server/routers/teams.js

```diff
@@ -0,0 +1,91 @@
+const express = require('express')
+const router = express.Router();
+const context = require('../entities/database/context')
+const Team = context.Team
+const User = context.User
+const apiError = require('../entities/api-error')
+
+router.get('/', async (req, res) => {
+
+    // get the team id
+    let name = req.params.name
+
+    if (name === null) {
+        try {
+            const allTeams = await Team.findAll()
+            res.status(200).send({ allTeams })
+        }
+        catch (err) {
+            res.status(400).send({ "message": "Something bad happened" })
+        }
+    }
+    else {
+        const team = await Team.findOne({
+            where: {
+                name
+            }
+        })
+        if (team === null) {
+            res.status(400).send({ "message": "Team not found" })
+        }
+        else {
+            res.status(200).send({ team })
+        }
+    }
+})
+
+router.post('/add', async (req, res) => {
```

Comment:
We should also rename this to `/`, as the HTTP methods are already dictating the operation (GET being the equivalent of READ, POST of CREATE and so on for all CRUD operations)

Reply:
Yes, I know, but I like giving them names :)

Reply:
Let's stick to good practices for this project though :)